### PR TITLE
Several fixes to PATH and PWD variables handling

### PR DESCRIPTION
**Pull request by paulstelian97**
See commit descriptions.

### CODE REVIEW

**Comment by diddledan** [comment]
Good work! Thank you :-)

**Comment by paulstelian97** [comment]
I'll push an extra commit now, PRE_NAMESPACE_PWD wasn't carried over correctly. Sorry for not testing.

**Comment by paulstelian97** [comment]
I think we're good now and I'll just wait for these to be merged.

File: ubuntu-wsl2-systemd-script.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ Defaults        env_keep += WSLENV
 Defaults        env_keep += WSL_INTEROP
 Defaults        env_keep += WSL_DISTRO_NAME
 Defaults        env_keep += PRE_NAMESPACE_PATH
+Defaults	env_keep += PRE_NAMESPACE_PWD
```
Comment (diddledan) [inline ubuntu-wsl2-systemd-script.sh:23]:
```suggestion
Defaults        env_keep += PRE_NAMESPACE_PWD
```

Nit-picking: the spacing here is with space characters, not tabs.. not important to be fixed, but a nice to have.

Comment (paulstelian97) [reply ubuntu-wsl2-systemd-script.sh:23]: Sure, I noticed it myself. In the projects I normally work with I see tabs, and in my terminal they aligned as well. Will fix.

Comment (paulstelian97) [reply ubuntu-wsl2-systemd-script.sh]: Fixed.

**Comment by DamionGans** [comment]
merged and thanks for your contributions

**Pull request merged**